stdlib/public/core/ManagedBuffer.swift

```diff
@@ -489,43 +477,15 @@ public func == <Header, Element>(
 /// This function is safe to use for `mutating` functions in
 /// multithreaded code because a false positive would imply that there
 /// is already a user-level data race on the value being mutated.
-public func isUniquelyReferencedNonObjC<T : AnyObject>(_ object: inout T) -> Bool
+public func isKnownUniquelyReferenced<T : AnyObject>(_ object: inout T) -> Bool
 {
   return _isUnique(&object)
 }
 
-internal func isUniquelyReferencedOrPinnedNonObjC<T : AnyObject>(_ object: inout T) -> Bool {
+internal func isKnownUniquelyReferencedOrPinned<T : AnyObject>(_ object: inout T) -> Bool {
```

Comment:
While you are changing this code, could you add an underscore to this internal function?